templates/profile/install.twig

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall hooks for the {{ profile }} Lightning
```

Comment:
I think this will read a bit funny in real-life use. Can it just say "for the {{ profile }} profile"?